multisense_ros/cfg/multisense.cfg

```diff
@@ -2,297 +2,343 @@
 PACKAGE = "multisense_ros"
 import os
 import roslib;roslib.load_manifest(PACKAGE)
+from enum import Enum
 
 if os.path.isfile('../../stack.xml'):
     from dynamic_reconfigure.parameter_generator import *
 else:
     from dynamic_reconfigure.parameter_generator_catkin import *
 
 class SensorConfig(object):
-    def __init__(self, name=None, sgm=False, motor=False, imu=False, lighting=False, aux=False, ar0234=False):
+    def __init__(self, name=None, sgm=False, motor=False, imu=False, lighting=False, aux=False, ar0234=False, features=False):
         self.name     = name
         self.sgm      = sgm
         self.motor    = motor
         self.imu      = imu
         self.lighting = lighting
         self.aux      = aux
         self.ar0234   = ar0234
+        self.features = features
     #enddef
 #endclass
 
+class SupportedFeatures(Enum):
+    NONE = 0
+    GROUND_SURFACE = 1
```

Comment:
Have you tried compiling this on older versions of ROS?

Reply:
Yep, compiles in noetic, melodic & kinetic